rust/connlib/tunnel/src/client.rs

```diff
@@ -281,37 +281,61 @@ fn send_dns_answer(
     }
 }
 
+/// A SANS-IO implementation of a client's functionality.
+///
+/// Internally, this composes a [`snownet::ClientNode`] with firezone's policy engine around resources.
```

Comment:
I did `cargo doc --no-deps --open -p firezone-tunnel` and this link didn't show up?

Reply:
I guess because snownet is not in the workspace

Reply:
You need to build the docs for the entire `rust/` directory. Then the links work.